Makefile

```diff
@@ -13,6 +13,7 @@ GOLANGCI_LINT_VERSION ?= v1.53.2
 # Arguments for getting directories & executing commands against them
 # PKG_RECEIVER_DIRS = $(shell find ./pkg/receiver/* -type f -name "go.mod" -print -exec dirname {} \; | sort | uniq)
 PKG_RECEIVER_DIRS = $(shell find ./pkg/receiver/* -type f -name '*go.mod*' | sed -r 's|/[^/]+$$||' |sort | uniq )
+CHECKS = prep lint-all metagen-all test-all genqlient-all tidy-all fmt-all
```

Comment:
do we need to run prep?

Reply:
Do these checks let you know if things have changed, or should they?

Reply:
Yes I think so, without the `test-all` command fails because it doesn't have necessary go module installed.  For example one of the packages it is missing without the prep command ```internal/scraper/githubscraper/generated-graphql.go:11:2: missing go.sum entry for module providing package github.com/Khan/genqlient/graphql (imported by github.com/liatrio/liatrio-otel-collector/pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper); to add:
        go get github.com/liatrio/liatrio-otel-collector/pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper```